core/src/main/java/org/elasticsearch/action/admin/cluster/snapshots/get/TransportGetSnapshotsAction.java

```diff
@@ -74,7 +75,7 @@ protected ClusterBlockException checkBlock(GetSnapshotsRequest request, ClusterS
         return state.blocks().globalBlockedException(ClusterBlockLevel.METADATA_READ);
     }
 
-    @Override
+    @Override @SwallowsExceptions(reason = "?")
```

Comment:
This is another place where we should probably have a common superclass that does the forwarding to the listener.